### PR TITLE
Improve text around RFC 7661

### LINKED ISSUE #149: Review RFC 7661 for multiplicative decrease section
I spotted only after re-reading the text later and already commented on the wg mailing list:

"the mechanisms described in [RFC7661] can be used to mitigate this
issue as it would allow using a value between _cwnd_ and
_flight_size_ to calculate the new _ssthresh_ in Figure 5."

AFAIK "using a value between _cwnd_ and _flight_size_ to calculate the new _ssthresh_" does not correctly capture the algo in RFC 7661. In the non-validated RFC 7661 uses

(Max(pipeACK,LossFlightSize))/2

and adjusts it with R (=retransmitted data) in the end of recovery phase.
And, in the validated phase RFC 7661 uses FlightSize/2.

It would be good to have Gorry to review this piece of text as the text seems to have been modified from what Gorry originally wrote.

### PR DESCRIPTION
**Pull request by larseggert**
Fixes #149

### CODE REVIEW

**Review by lisongxu — DISMISSED** [review]
Thanks

File: draft-ietf-tcpm-rfc8312bis.md [hunk 1]
```diff
@@ -674,33 +674,36 @@ The parameter {{{β}{}}}*<sub>cubic</sub>* SHOULD be set to 0.7, which
...
+use *cwnd* MUST use other measures to prevent *cwnd* from growing when
+the volume of bytes in flight is smaller than *cwnd*. This also
+effectively avoids *cwnd* from growing beyond the receive window. Such
+measures are important to prevent a CUBIC sender from using an
+arbitrarily high cwnd *value* when calculating new values for
+*ssthresh* and *cwnd* when congestion is detected. This might not be as
+robust as the mechanisms described in {{?RFC7661}}.
+
+A QUIC sender that uses *cwnd* to calculate new values for *cwnd* and
+*ssthresh* after detecting a congestion event is REQUIRED to apply
```
Comment (goelvidhi) [inline draft-ietf-tcpm-rfc8312bis.md:705]: is REQUIRED (in caps) a known term?

Comment (larseggert) [reply draft-ietf-tcpm-rfc8312bis.md:705]: Yes, see RFC 2119. Same as MUST.

File: draft-ietf-tcpm-rfc8312bis.md [hunk 2]
```diff
@@ -674,33 +674,36 @@ The parameter {{{β}{}}}*<sub>cubic</sub>* SHOULD be set to 0.7, which
...
-for the congestion window and slow-start threshold on a congestion
-event is required to apply similar mechanisms {{!RFC9002}}.
+could easily encounter notable variations in the volume of data sent
+from one RTT to another, resulting in *flight_size* that is
+significantly less than *cwnd* when there is a congestion event. The
+congestion response would therefore decrease *cwnd* to a much lower
+value than necessary. To avoid such suboptimal performance, the
+mechanisms described in {{?RFC7661}} can be used. These describe how
+to calculate a new *cwnd* for sending and after congestion, a
+new *ssthresh* with a value between the *cwnd* and *flight_size*. The
```
[the remark this answers is not in the document]
Comment (gorryfair) [reply draft-ietf-tcpm-rfc8312bis.md:688]: Would it be better to state:

"These describe how to manage and use the  *cwnd* and *ssthresh* during a rate-limited Interval, and how to update *cwnd* and *ssthresh*  after congestion has been detected.

Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:688]:
```suggestion
mechanisms described in {{?RFC7661}} can be used. These describe how
to manage and use *cwnd* and *ssthresh* during a rate-limited Interval,
and how to update *cwnd* and *ssthresh* after congestion has been detected. The
```
Diff of @gorryfair's suggestion - OK to merge?

Comment (gorryfair) [reply draft-ietf-tcpm-rfc8312bis.md:688]: Works for me.

[the remark this answers is not in the document]
Comment (goelvidhi) [reply draft-ietf-tcpm-rfc8312bis.md:688]: Yes, I like this text.

Comment (goelvidhi) [reply draft-ietf-tcpm-rfc8312bis.md:688]: works for me as well.

**Review by goelvidhi — DISMISSED** [review]

**Pull request merged**